projects/plugins/paypal-payment-buttons/readme.txt

```diff
@@ -1,70 +1,90 @@
+3. Choose between Stacked Buttons and Single Buttons based on your needs.
+   - **Stacked Buttons (Recommended)**: This option lets you present all of your product information and PayPal payment method upfront on your website.
+   - **Single Buttons**: This option lets you quickly paste a single button on your site, with no product information.
+4. Copy the provided button code from the PayPal site.
 
-You may provide arbitrary sections, in the same format as the ones above.  This may be of use for extremely complicated
-plugins where more information needs to be conveyed that doesn't fit into the categories of "description" or
-"installation."  Arbitrary sections will be shown below the built-in sections outlined above.
 
-== A brief Markdown Example ==
+#### If you are using the Stacked Buttons:
 
-Ordered list:
+1. Select the HTML code language from the dropdown above your button code. 
+2. Then paste the code for the `<head>` into the first text box in the PayPal Payment Buttons block. (We’ll take care of placing this code into your post or page `<head>` only once.)
+3. Then paste the `<body>` code into the second text box in the PayPal Payment Buttons block.
 
-1. Some feature
-1. Another feature
-1. Something else about the plugin
+#### If you are using the Single Button:
 
-Unordered list:
+1. Click the Single Button option in the PayPal Payment Buttons block.
+2. Copy the single button code from the PayPal site and paste it into the text area of the block.
 
-* something
-* something else
-* third thing
+By repeating the process above, you can add as many PayPal Payment Buttons blocks to your page as you like.
 
-Here's a link to [WordPress](https://wordpress.org/ "Your favorite software") and one to [Markdown's Syntax Documentation][markdown syntax].
-Titles are optional, naturally.
+### Taking Payments with PayPal
+
+1. When a visitor clicks a PayPal payment button, a new window will open to guide them through the PayPal checkout process.
+2. Visitors can use an existing PayPal account or a credit or debit card to complete their purchase.
+3. All payments are credited to the PayPal business account that was used to create the payment button code.
+
+== Installation ==
 
-[markdown syntax]: http://daringfireball.net/projects/markdown/syntax
-"Markdown is what the parser uses to process much of the readme file"
+1. Upload the PayPal Payment Buttons plugin to your WordPress site.
+2. Activate the plugin through the 'Plugins' screen in WordPress.
+3. Add the PayPal Payment Buttons block to any post or page using the block editor.
+4. Configure your PayPal account settings within the block to start accepting payments.
 
-Markdown uses email style notation for blockquotes and I've been told:
-> Asterisks for *emphasis*. Double it up  for **strong**.
+== Frequently Asked Questions ==
+
+= Can I test with a sandbox account? =
```

Comment:
This is currently an FAQ. I thought that it felt weird in the description.

<img width="1388" height="712" alt="CleanShot 2025-07-31 at 21 24 00@2x" src="https://github.com/user-attachments/assets/400fc622-886d-489b-b21a-8f33576fa5bb" />

I'm open to feedback though.